_posts/2019-10-27-on-code-reviews.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+permalink: on-code-reviews
+title: 'On Code Reviews'
+tags: [programming]
+---
+
+I wanted to share some insights into how I review code, learning from others,
+helping others, and getting a better overview of the code base which ultimately
+allows me to write better[^1] code. This post is inspired by a recent
+[tweet](https://twitter.com/kristoferbaxter/status/1186290473322741760), a]
+similar [blog post](https://techbeacon.com/app-dev-testing/10-commandments-navigating-code-reviews)
+(by [Angie Jones](https://twitter.com/techgirl1908)) and a [talk](https://www.youtube.com/watch?v=Ea8EiIPZvh0)
+on the topic (by [April Wensel](https://twitter.com/aprilwensel)) from 2018's
+try! Swift NYC.
+
+- **I point out the good things.** This goes for anything really; not just code
+reviews. If there's something you like or learned, let the author know. I
+believe this will make a big difference on how a review is perceived, and is a
+win-win all around.<br />
```

Comment:
Remove `<br/>`

Reply:
I wanted to make sure there's a line break there 😄

Reply:
Hmm ok I didn’t know that you can use html tags in markdown.

Reply:
Markdown will end up just being html, so you can always do that! The same goes for the `<sub></sub>` at the bottom of the post :)